flipperdevices/flipperzero-firmware
# fbt: support for LOADER_AUTOSTART; post-build size stats

# What's new

- Restored support for LOADER_AUTOSTART. Set `./fbt LOADER_AUTOSTART=NFC` to run NFC app on system boot
- Added fwsize.py script designed for extraction of elf section sizes and binary flash size in pages

# Verification 

- Build firmware with LOADER_AUTOSTART=NFC and check that it's ran on system boot
- Check out console output for new post-build stats. Ex.: 
```
...
Firmware size
.text         579768 (566.18 K)
.rodata       129896 (126.85 K)
.data           1404 (  1.37 K)
.bss            7956 (  7.77 K)
.free_flash   337084 (329.18 K)
        HEX     build\f7-firmware-C\firmware.hex
Setting build\f7-firmware-C as latest built dir (./build/latest/)
        BIN     build\f7-firmware-C\firmware.bin
firmware.bin:  174 pages (last page 70.41% full)
```

# Checklist (For Reviewer)
